controls/def.cf

```diff
@@ -126,6 +126,14 @@ bundle common def
                           not(isvariable("trustkeysfrom"))),
         meta => { "defvar" };
 
+      "cf_serverd_maxconnections"
```

Comment:
So I'm wondering, shouldn't we try to nest these in `def.json`? So the top-level key is `cf_serverd` and then under that we have `maxconnections`? Or even `controls -> server -> maxconnections`? The `def.json` parser could definitely connect `controls -> AGENTNAME -> SETTING` with `body AGENTNAME control { SETTING => ... }` so we don't have multiple names for the same setting, so we could automatically have **all** the controls easily available, and so they can simply be set without policy.